src/pages/index.tsx

```diff
@@ -31,6 +36,38 @@ const Home: NextPage = () => {
             </p>
           </div>
         </Link>
+
+        <button
```

Comment:
create a components folder in src. inside that create a file called ThemeToggle.tsx and move this button there (/src/components/ThemeToggle.tsx)

Now, create an index.ts file inside the components folder and export the component there like this ->

```ts
export { ThemeToggle } from "./ThemeToggle";
```

Then import it here like this ->
```tsx
import { ThemeToggle } from "./components"
```

This is for component reusability

Reply:
applied @AyanavaKarmakar